base/strings/basic.jl

```diff
@@ -34,6 +34,7 @@ If you need to subsequently modify `v`, use `String(copy(v))` instead.
 """
 String(v::Array{UInt8,1})
 
+String(x::Symbol) = string(x)
```

Comment:
Then we should merge this first, and make another PR to replace it with `convert`?